BusBoard/.idea/.idea.BusBoard/.idea/.gitignore

```diff
@@ -0,0 +1,13 @@
+﻿# Default ignored files
```

Comment:
Could you please add a `.gitignore` file. It leaves out configuration files and packages, so that the pull requests only include code changes that you've actually made